server/views/components/work-media/work-media.njk

```diff
@@ -3,6 +3,7 @@
     {% if model.queryString %}
       <div class="flush-container-left {{ {s:2} | spacingClasses({padding: ['top']}) }}">
         <a href="/works{{ model.queryString }}#{{model.id}}"
+          data-track-event='{"category": "work", "action": "return-to-results", "label": "query:{{ model.queryString }}, id:{{ model.id }}"}'
```

Comment:
Why do we have `id` on here? It doesn't get tracked from GA.
Can we run this past @hthair too?

Reply:
`"query:{{ model.queryString }}, id:{{ model.id }}"` is one string inside the `label`.

Yep – waiting on a time to show it to @hthair

Reply:
@davidpmccormick I'm free at 4.30 or if planning finishes early